tardis/model.py

```diff
@@ -90,21 +91,24 @@ def __init__(self, tardis_config):
 
         self.t_inner = tardis_config.plasma.t_inner
 
+        self.v_inner = tardis_config.structure.v_inner
+        self.v_outer = tardis_config.structure.v_outer
+        self.v_middle = 0.5 * (self.v_inner + self.v_outer)
```

Comment:
It's just a small thing which isn't important here. But in general one should prefer `0.5 * x + 0.5 * y` to prevent overflowing.